### ISSUE TITLE
Does not work. The update doesn't help.

### ISSUE THREAD

**Issue by nvjob** [sashaok123/LoRA-EXTRACTOR#10]
Extraction:
Traceback (most recent call last):
  File "D:\ai\LoRA-EXTRACTOR\lib\extract_LoRA_from_models.py", line 10, in <module>
    import library.model_util as model_util
ModuleNotFoundError: No module named 'library'

**Comment by nvjob** [comment]
Should be added to the description.
What you need to download separately https://github.com/kohya-ss/sd-scripts all files in the lib folder.

**Comment by divinebaboon** [comment]
Yeah this repo needs some work, for me, I had to change a few lines for it to work,
extract_LoRA_from_models.py, line 10 -> import model_util
lora.py, line 11 -> import train_util
train_util.py, line 31 -> import model_util

**Comment by Rb-diff** [comment]
well that soundw like a good solution, except... the sd scripts download doesn't have a lib folder. It has a Library folder which in that case... doesn't help because I've tried that.

**Comment by LoFiApostasy** [comment]
Fresh install, everything works. (Windows.)
Ran into the error "RuntimeError: mat1 and mat2 shapes cannot be multiplied" which just means your multiplier is set too high.  Try the default or 512, 768, etc...
Thanks!
